scripts/hyper_tuning.py

```diff
@@ -40,28 +40,24 @@
     if args.early_stopping_delta:
         early_stopping_delta = args.early_stopping_delta
 
-    if args.use_rgb:
```

Comment:
Brauchen wir das in der `training.py` eigentlich noch? Das wird meiner Ansicht nach nirgendwo verwertet...